pyproject.toml

```diff
@@ -95,29 +87,36 @@ test = [
     "pytest~=7.4.2",
     "pytest-cov~=4.0.0",
     "pytest-asyncio~=0.20.3",
-    "looptime~=0.2",
+    "looptime~=0.2.0",
     "coverage[toml]~=6.5.0",
 ]
-build = [
-    "wheel~=0.40.0",
-    "build~=0.10.0",
-    "twine~=5.1.1",
+docs = [
+    "sphinx==7.0.1",
+    "sphinxcontrib-trio~=1.1.2",
+    "sphinx-hoverxref==1.3.0",
+    "sphinx-autobuild~=2021.3",
+    "sphinxcontrib-towncrier==0.3.2a0",
+    "towncrier==23.6.0",
+    "sphinx-notfound-page==0.8.3",
 ]
 
-
-[tool.pdm]
-# Ignore `requires-python` warnings when locking, the latest versions of some
-# dependencies already require >=3.9
-# See also https://pdm-project.org/en/latest/usage/config/#ignore-package-warnings
-ignore_package_warnings = ["*"]
-
-[tool.pdm.scripts]
-docs = { cmd = "nox -Rs docs --", help = "Build the documentation for development" }
-lint = { cmd = "nox -Rs lint --", help = "Check all files for linting errors" }
-pyright = { cmd = "nox -Rs pyright --", help = "Run pyright" }
-setup_env = { cmd = "{pdm} install -G:all", help = "Set up the local environment and all dependencies" }
-post_setup_env = { composite = ["python -m ensurepip --default-pip", "pre-commit install --install-hooks"] }
-test = { cmd = "nox -Rs test --", help = "Run pytest" }
+[tool.uv]
+required-version = ">=0.8.4"
+default-groups = "all"
+# temporary: keep version compat with pdm locking
+fork-strategy   = "fewest"
```

Comment:
How bad of an idea would it be to remove this? :happythonk:

Reply:
This was necessary until the aiohttp fix for deprecated timeouts was added. This causes resolution to be the same version across all python versions, eg aiohttp 3.10 across every single version of python.

Removing or updating this line will cause aiohttp to be resolved as 3.12 on all supported platforms which IS what we want.

Reply:
https://github.com/DisnakeDev/disnake/pull/1322/commits/40f7750e74929551cb94499cc67aa6707ced1bd7